libs/libedit/Makefile

```diff
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libedit
-PKG_VERSION:=20240517-3.1
+PKG_VERSION:=20240808-3.1
```

Comment:
Looks like the change will have to be to `3.1-r20240808`.
Do you know how it is with version comparisons? Because the opkg apparently considers the new version number older than the original (see `opkg compare-versions 20240808-3.1 '<' 3.1-r20240808` returns `false`)

Reply:
I'm not sure about that sorry

Reply:
I think there are multiple options:
-  Inform users about this and that they should do a manual "downgrade" (really upgrade)
- Let them use `auc` or `owut` for an upgrade, the sys upgrade tools will inform the user about the package downgrade but the user likely understand the situation.
- Set the version to `20240808.3.1` which should be compatible with both

Reply:
> Set the version to 20240808.3.1

That would then require additional Makefile variables, as the version in the original format is part of the upstream filename.

We might have an APK related wikipage with helpful hints about solutions to that kind of needs. And the new CI message might link to that page.